website/docs/r/storage.html.md

```diff
@@ -48,7 +48,7 @@ The following arguments are supported:
 
     * `password_type` - (Optional) (one of plain, crypt) Required if password is set (ignored for private templates and public Windows templates).
 
-    * `sshkeys` - (Optional) (array of any - minItems: 0) Public Linux templates only! The UUIDs of SSHkeys to install for the root user.
+    * `sshkeys` - (Optional) (array of any - minItems: 0) Public Linux templates only! The UUIDs of SSH keys to install for the root user.
```

Comment:
The UUIDs of SSH keys to be added for the root user.

Reply:
if we nitpick about this, imho it should be ssh keys not SSH ;)

Reply:
updated

Reply:
@fkr not sure. Lots of sites (e.g. [GitHub's help](https://help.github.com/en/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account)) use all-uppercase for secure shell abbreviation (as it actually be the case in English language? No idea.)

Reply:
actually @itakouna and you are correct. Even all the SSH manual pages use SSH. thanks for pointing this out!